regression/ansi-c/gcc_attributes11/main.c

```diff
@@ -0,0 +1,8 @@
+static int __attribute__ ((__section__(".init.text"))) __attribute__((no_instrument_function)) dp83640_init(void) { return 0; }
+int init_module(void) __attribute__((alias("dp83640_init")));;
+
+int main()
+{
+  dp83640_init();
```

Comment:
Cite the source for this (presumably some particular Linux kernel file?)

Reply:
Done.